Проблема: `htmlacademy/req-webp-in-picture` требует webp от вектора

### Ошибка из терминала

```shell
File: build/index.html
Config file: ./.linthtmlrc
 21:3  error  Element "picture" must contain a "source" child with a "type" attribute containing "webp".  htmlacademy/req-webp-in-picture 

✖ 1 problems (1 errors, 0 warning)
```

### Часть кода на котором произошла ошибка. Какая конфигурация использовалась

```html
<picture>
	<source type="image/svg+xml" media="(width >= 1280px)" srcset="images/logo-1280.svg" width="33" height="38">
	<source type="image/svg+xml" media="(width >= 768px)" srcset="images/logo-768.svg" width="33" height="38">
	<img src="images/logo.svg" srcset="images/logo.svg" width="33" height="38" alt="Знак Cat Energy">
</picture>
```

### Описание проблемы

Правило `htmlacademy/req-webp-in-picture` требует webp в `picture`, который нужен не только для растра.
Хотя на практике в рабочих проектах я ни разу не встречал такой адаптации логотипов, да и кроме логотипов svg внутри `picture` никогда не использовал, но чтобы на 2 курсе по вёрстке такое требование включать, надо сначала логотип (вернее фирменный знак) сделать одинаковым на всех вьюпортах (оставить адаптацию только размеров).

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.0.25
+Fixes `req-webp-in-picture` to not check `<picture>` if all `<source>` have attribute `type="image/svg+xml"`. 
+
 ## 1.0.24
 - Adds a `req-tags-presence` rule that requires the specified tags on the page.
 - Adds a `req-preload-font` rule that requires the `preload` value for the font.

--- a/package.json
+++ b/package.json
@@ -19,6 +19,6 @@
   "homepage": "https://github.com/htmlacademy/linthtml-config-htmlacademy#readme",
   "dependencies": {
     "@linthtml/linthtml": "0.9.6",
-    "linthtml-rules-htmlacademy": "1.0.20"
+    "linthtml-rules-htmlacademy": "1.0.21"
   }
 }

--- a/test/catalog.html
+++ b/test/catalog.html
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>HTML Academy: Девайс. Каталог</title>
     <link href="./css/style.min.css" rel="stylesheet">
+    <link rel="preload" href="./fonts/oswald/oswald-regular.woff2" type="font/woff2" as="font">
   </head>
   <body class="catalog-page">
     <header class="header header--catalog-page">

--- a/test/index.html
+++ b/test/index.html
@@ -420,6 +420,12 @@
       <svg width="20" height="20"></svg>
     </div>
 
+    <picture>
+      <source type="image/svg+xml" media="(width >= 1280px)" srcset="images/logo-1280.svg" width="33" height="38">
+      <source type="image/svg+xml" media="(width >= 768px)" srcset="images/logo-768.svg" width="33" height="38">
+      <img src="images/logo.svg" srcset="images/logo.svg" width="33" height="38" alt="Знак Cat Energy">
+    </picture>
+
     <script src="./js/script.min.js"></script>
   </body>
 </html>

